ios/BrightcovePlayer.m

```diff
@@ -223,7 +224,7 @@ - (void)playbackController:(id<BCOVPlaybackController>)controller playbackSessio
                 @"mediainfo": mediainfo
             });
         }
-        if (_autoPlay) {
+        if (_autoPlay && _playing) {
```

Comment:
This doesn't make a lot of sense.

If its autoplay, and it's already playing? Can you please double check it so we don't have any regression?